docs/faq.md

```diff
@@ -0,0 +1,7 @@
+## Frequently Asked Questions
```

Comment:
Let's add to the main `README.yaml` since this file will not be easily discovered